client/components/marketing-survey/cancel-purchase-form/index.jsx

```diff
@@ -461,9 +461,9 @@ class CancelPurchaseForm extends React.Component {
 		);
 	};
 
-	openCalendly = () => {
-		this.props.clickCalendly();
-		return window.open( 'https://calendly.com/wordpressdotcom/wordpress-com-business-site-setup/' );
+	openConcierge = () => {
+		this.props.clickConcierge();
+		return window.open( 'https://wordpress.com/me/concierge' );
```

Comment:
Could we consider to use a relative path here? By `window.open( '/me/concierge' )`, we can make sure we stay at the same origin, whether its `calypso.local` or `wpcalypso.wordpress.com`.

Reply:
Great point, updated.